client/landing/subscriptions/tabs-switcher/tabs-switcher.tsx

```diff
@@ -18,8 +18,8 @@ type TabsSwitcherProps = {
 
 const getRoute = ( baseRoute: string, path: string ): string => {
 	// This is a temporary redirect to the re-skinned Subscription Management portal.
-	// Once the new "Commnets" and "Sites" views are ready, this will be removed.
-	const temporaryRedirect = [ 'comments', 'sites' ];
+	// Once the new "Commnets" views is ready, this will be removed.
+	const temporaryRedirect = [ 'comments' ];
```

Comment:
Do we need to check for specially EN & external users (well it's always external users now) before deciding whether to include or omit `'sites'`?

Reply:
Is Calypso Settings already enabled to other user locales? If so, yes, we should check it. If not, the user would just be on Calypso if it's an EN user.

Reply:
Only the EN users should end up on the Calypso Settings page at the moment. This means that we shouldn't need additional check for the user's locale.

I have tested this with `pt-BR` now: The current Reskinned portal's "Settings" tab sends the user to the Calypso Settings page only when the locale is EN:

![Markup on 2023-04-04 at 14:20:04](https://user-images.githubusercontent.com/25105483/229789712-7f92a706-ebb1-49a1-a744-70826c651469.png)

Otherwise it stays in the Reskinned portal:

![Markup on 2023-04-04 at 14:19:02](https://user-images.githubusercontent.com/25105483/229789747-943d1e13-8bd3-42e8-b11a-0f52d870eb7a.png)